packages/Button/src/Button.styles.js

```diff
@@ -230,7 +230,7 @@ const buttonStyles = props => `
   ${commonStyles}
   ${sizeStyles[props.size]}
   ${kindStyles(props)[props.kind]}
-  ${props.isFullWidth && !props.isSquare ? fullWidthStyles : ""}
+  ${props.isFullWidth ? fullWidthStyles : ""}
```

Comment:
@mikrotron I don't see that the `isSquare` prop is necessary anymore, or there another reason it is still existing here. The `isFullWidth` prop should be enough to apply the `fullWidthStyles`?

Reply:
It seems like we removed the `isSquare` prop, maybe when it was migrated from `acl-ui`:
https://github.com/acl-services/paprika/blob/master/packages/Button/src/Button.js

So yeah, this is a relic, thanks for cleaning it up.

Reply:
This is a `Patch` update to `<Button>` though, which should be indicated in the description, please.

Reply:
done